public/video-ui/src/actions/PlutoActions/getProjects.ts

```diff
@@ -0,0 +1,40 @@
+import { Dispatch } from 'redux';
+import { KnownAction } from '../../actions';
+import { getPlutoProjects, PlutoProject } from '../../services/PlutoApi';
+
+function requestProjects(): KnownAction {
+  return {
+    type: 'PLUTO_PROJECTS_GET_REQUEST',
+    receivedAt: Date.now()
+  };
+}
+
+function receiveProjects(plutoProjects: PlutoProject[]): KnownAction {
+  return {
+    type: 'PLUTO_PROJECTS_GET_RECEIVE',
+    receivedAt: Date.now(),
+    projects: plutoProjects
+  };
+}
+
+function errorReceivingProjects(error: unknown): KnownAction {
+  return {
+    type: 'SHOW_ERROR',
+    message: 'Could not get Pluto Projects',
+    receivedAt: Date.now(),
+    error: error
+  };
+}
+
+export function getProjects({ commissionId }: { commissionId: string }) {
```

Comment:
It might be simpler if this function just took `commissionId` as the argument and not an object.